scripts/orderlywebconfig.properties

```diff
@@ -0,0 +1,7 @@
+app.url=https://localhost/reports
+auth.fine_grained=false
+montagu.api_url=https://172.17.0.1/api/v1
+montagu.client_api_url=https://localhost/api/v1
```

Comment:
..and we need this because https://172.17.0.1/api/v1 won't work from the browser. You may also need to inveigle your browser into allowing calls to local self-signed api, e.g. chrome://flags/#allow-insecure-localhost